### ISSUE TITLE
Feature: Allow method importing

### ISSUE THREAD

**Issue by dwilt** [anarchicknight/react-native-communications#24]
Right now, we have to do:

```javascript
import Communications from 'react-native-communications';

// Communications.email()
// Communications.text()
```
it would be nice to be able to import the exact methods we want to use:

```javascript
import {
  text,
  email
} from 'react-native-communications';

// email()
// text()
```
to be honest, I'm not sure why this already isn't working as the `react-native-communications` has `export default communication` at the bottom and `communication` is an object with the methods on it..

**Comment by anarchicknight** [comment]
will hopefully get a chance to look at this on the weekend

**Comment by dwilt** [comment]
I actually think I might be able to fix this myself. Cool if I put in a PR?

**Comment by anarchicknight** [comment]
yeah go for it :)

**Comment by dwilt** [comment]
I'm new to this process. Checked out the repo, made the changes, but can't submit the feature branch for a PR. Says I need permissions?

**Comment by anarchicknight** [comment]
the top answer on this link may help. You want to start off by forking the repo rather than checking it out

http://stackoverflow.com/questions/14680711/how-to-do-a-github-pull-request

**Comment by anarchicknight** [comment]
resolved in 2.1.1 which has just been published
